### PR TITLE
General fixes to links across site

### PR DESCRIPTION
**Pull request by micpap25**
Fixes broken / odd links that I ran into while viewing the site.

### CODE REVIEW

**Review by ISSOtm — CHANGES_REQUESTED** [review]
I overall approve, but I would prefer if the links mentioned something more general than "Discord server", since we offer more than that.

File: list/list.md [hunk 1]
```diff
@@ -14,12 +14,12 @@ A curated list of awesome Game Boy (Color) Development resources, tools, docs, r
...
 
 <div style="letter-spacing: -0.05em; font-weight: 500; font-size: 18px;">
-This project is open source and community-lead. Come <a href="#contribute">contribute! </a></div>
+This project is open source and community-lead. Come <a href="https://gbdev.io/contribute">contribute! </a></div>
 
 </div>
 <br>
 <div style="letter-spacing: -0.03em; font-weight: 500; font-size: 20px;">
-<a href="https://discord.gg/gpBxq85">Join us on Discord!</a></div>
+<a href="https://gbdev.io/chat">Join the community!</a></div>
```
Comment (micpap25) [inline list/list.md:22]: Does this sound better?

Comment (ISSOtm) [reply list/list.md:22]: Or perhaps "chat with the community"?

File: list/README.md [hunk 2]
```diff
@@ -60,11 +60,11 @@ The single, most comprehensive technical reference to Game Boy available to the
...
+<p ><a class="projectTitle" target="_blank" href="https://github.com/gbdev/rgbds">RGBDS&nbsp;</a> <a class="github-button" target="_blank" href="https://github.com/gbdev/rgbds" data-icon="octicon-star" data-show-count="true" aria-label="Star gbdev/rgbds on GitHub">Star</a>
 <span></span></p>
 Rednex Game Boy Development System: the de-facto development toolkit for the Game Boy and Game Boy Color.
 <br><br>
-<p class="projectTitle"><a target="_blank" href="https://gbdev.io/chat">Discord server</a>&nbsp;<img height="22" src="https://img.shields.io/badge/dynamic/json.svg?label=chat&colorB=green&suffix=%20online&query=presence_count&uri=https://discordapp.com/api/guilds/303217943234215948/widget.json&style=flat-square"></p>
+<p class="projectTitle"><a href="https://gbdev.io/chat">Discord server</a>&nbsp;<img height="22" src="https://img.shields.io/badge/dynamic/json.svg?label=chat&colorB=green&suffix=%20online&query=presence_count&uri=https://discordapp.com/api/guilds/303217943234215948/widget.json&style=flat-square"></p>
```
Comment (ISSOtm) [inline list/README.md:67]: (Note: once [this thread](https://github.com/gbdev/gbdev.github.io/pull/29/files#r615426516) is resolved, the same should be applied here.)

File: list/README.md [hunk 3]
```diff
@@ -60,12 +60,12 @@ The single, most comprehensive technical reference to Game Boy available to the
...
 <span></span></p>
 Rednex Game Boy Development System: the de-facto development toolkit for the Game Boy and Game Boy Color.
 <br><br>
-<p class="projectTitle"><a target="_blank" href="https://gbdev.io/chat">Discord server</a>&nbsp;<img height="22" src="https://img.shields.io/badge/dynamic/json.svg?label=chat&colorB=green&suffix=%20online&query=presence_count&uri=https://discordapp.com/api/guilds/303217943234215948/widget.json&style=flat-square"></p>
-Where our community thrives. Here we <b>chat</b>, discuss, help each other and show what we are working on. There's an IRC channel, too.
+<p class="projectTitle"><a href="https://gbdev.io/chat">Chat Channels</a>&nbsp;<img height="22" src="https://img.shields.io/badge/dynamic/json.svg?label=chat&colorB=green&suffix=%20online&query=presence_count&uri=https://discordapp.com/api/guilds/303217943234215948/widget.json&style=flat-square"></p>
+The places our community thrives. Here we <b>chat</b>, discuss, help each other and show what we are working on. We have a Discord server, an IRC channel, and more.
```
Comment (micpap25) [inline list/README.md:68]: Updated the description here as well to highlight the multiple platforms. While on the topic, is the bolding on `chat` necessary? It seems a bit odd to highlight.

**Pull request merged**